src/v/raft/consensus.h

```diff
@@ -551,12 +545,17 @@ class consensus {
      * Hydrate the consensus state with the data from the snapshot
      */
     ss::future<> hydrate_snapshot();
-    ss::future<> do_hydrate_snapshot(storage::snapshot_reader&);
+
+    void load_from_metadata(const snapshot_metadata&);
```

Comment:
nit: I would prefer a word "snapshot" somewhere in the function name (there are many kinds of metadata :)), also it doesn't fully load metadata, just updates some offsets. `update_offsets_from_snapshot` maybe?

Reply:
Done